jekyll/_cci1/docker.md

```diff
@@ -5,13 +5,17 @@ categories: [how-to]
 description: Continuous Integration and Delivery with Docker
 ---
 
-CircleCI currently offers beta support for running Docker within build containers. Docker is an extremely flexible tool that supports many different use cases. This article attempts to address several of the most popular uses for Docker on CircleCI, but it is not an exhaustive list. 
+**CircleCI 2.0 is now in Beta with native Docker support. Please see the [Beta access page](https://circleci.com/beta-access/) and the [2.0 Documentation](https://circleci.com/docs/2.0/).**
+
+CircleCI 1.0 offers support for running Docker within build containers. Support for Docker on CircleCI 1.0 is limited to Docker up to version 1.9 and has several limitations. **If you're starting a new project that uses Docker we strongly recommend using CircleCI 2.0. This documentation is for customers using Docker for existing projects on CircleCI 1.0.**
+
+---
+
+## Using Docker on CircleCI 1.0
```

Comment:
This might just be me, but are we worried about having to namespace all of these headers with the version of CircleCI? Are we forced to do this because of the way search works?